recipe/recipe.yaml

```
@@ -0,0 +1,48 @@
+schema_version: 1
+
+context:
+  name: click
+  version: 8.3.1
+  python_min: "3.10"
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/click-${{ version }}.tar.gz
+  sha256: 12ff4785d337a1bb490bb7e9c2b1ee5da3112e94a8622f26a6c77f5d2fc6842a
+
+build:
+  noarch: python
+  number: 1
+  script: python -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - flit-core <4
+    - pip
+  run:
+    - python >=${{ python_min }}
+    - colorama
+    - if: win
+      then: __win
```

I think we need a corresponding:
```
```suggestion
    - if: win
      then: __win
    - if: unix
      then: __unix
```